src/SplitPane.coffee

```diff
@@ -0,0 +1,72 @@
+React = require 'react'
+H = React.DOM
+Pane = require './Pane'
+Divider = require './Divider'
+ReactDOM = require 'react-dom'
+
+module.exports = class SplitPane extends React.Component
```

Comment:
Needs comments and `@propTypes` to describe what props it takes and thus how to use it.